src/AppInstallerCLICore/Workflows/WorkflowBase.cpp

```diff
@@ -410,21 +423,22 @@ namespace AppInstaller::CLI::Workflow
             {
                 Logging::Telemetry().LogMultiAppMatch();
 
-                if (WI_IsFlagSet(context.GetFlags(), Execution::ContextFlag::InstallerExecutionUseUpdate))
+                if (WI_IsFlagSet(context.GetFlags(), Execution::ContextFlag::CompositeInstalledSource))
                 {
                     context.Reporter.Warn() << Resource::String::MultipleInstalledPackagesFound << std::endl;
+                    context << ReportListResult(false);

```

Comment:
>context << ReportListResult(false); [](start = 20, length = 35)

I don't think we want this.  See `ReportListResult` and `IPackage::GetProperty` and create another method to report these with only Name and Id; I don't think version is very important here, and both cases are used to simply give the options of things that were found.